yarn.lock

```diff
@@ -12554,7 +12582,7 @@ fuse.js@^3.4.6:
   resolved "https://registry.yarnpkg.com/fuse.js/-/fuse.js-3.4.6.tgz#545c3411fed88bf2e27c457cab6e73e7af697a45"
   integrity sha512-H6aJY4UpLFwxj1+5nAvufom5b2BT2v45P1MkPvdGIK8fWjQx/7o6tTT1+ALV0yawQvbmvCF0ufl2et8eJ7v7Cg==
 
-gaba@^1.11.0, gaba@^1.9.3:
+gaba@^1.9.3:
```

Comment:
Huh, well this is unfortunate. We should update this on `rpc-cap` as well, to get rid of this second copy.

Reply:
Yup, a forthcoming set of PRs